src/main/java/io/stargate/sgv2/jsonapi/service/embedding/operation/CohereEmbeddingClient.java

```diff
@@ -33,16 +34,19 @@ public class CohereEmbeddingClient implements EmbeddingProvider {
   private String modelName;
   private String baseUrl;
   private final CohereEmbeddingProvider embeddingProvider;
+  private Map<String, Object> vectorizeServiceParameter;
 
   public CohereEmbeddingClient(
```

Comment:
Asking offline; but question is whether concrete impl clients are needed on Data API side -- perhaps this is only needed temporarily? Otherwise it seems like impl differences should only be handed on Embedding Gateway service side.